assets/configuration.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+    if (basename(__FILE__) != basename($_SERVER["SCRIPT_FILENAME"])) {
+        
+        /* CONFIGURATION */
+        
+        if (!isset($debug)) {
+            $debug = true; // Enable PHP error_reporting
+        }
+        
+        /* TIME DEFINITIONS */
+        
+        define("TIME_SECOND", 1);
+        define("TIME_MINUTE", 60);
+        define("TIME_HOUR", 60 * 60);
+        define("TIME_DAY", 24 * 60 * 60);
+        define("TIME_WEEK", 7 * 24 * 60 * 60);
+        
+        //$language = "en_US"; // Explicitly set language locale. Remove or comment out to auto detect.
+        
+        /*
+            This requires PHP Locale class.
+            sudo apt-get install php-intl
+            
+            https://stackoverflow.com/questions/18346531/how-to-enable-php-locale-class
+            
+            Other languages may have rogue locales.
+            https://superuser.com/questions/1519501/what-exactly-do-the-three-special-locales-called-en-us-posix-en-001-and
+            
+        */
+            
+        $maintenance = false;
+        
+        $privateFolder = "/private/spiteful-chat"; // Private directory location from directory behind DOCUMENT_ROOT
+        $composerFolder = "./vendor"; // PHP Composer Directory
+        $nodeModulesFolder = "./node_modules"; // NPM Modules Directory
+        $spiteSocketServerHost = "127.0.0.1"; // Spiteful Server Websocket Host
+        $spiteSocketServerPort = "12345"; // Spiteful Server Websocket Port
+        
+        $loginSessionLength = TIME_HOUR * 8; // How long the session should last
+        
+        $googleDataClientId = "13964511690-v18rnirom2pk4ct8rc5careg7geda2mg.apps.googleusercontent.com"; // x.apps.googleusercontent.com
+        $googleDataLoginUri = "https://dleovl.me/spiteful-chat/cwg";
+        $googleSigninClientId = "905753697514-bat1tv7nld4ltb4fhfo9op5gmf93jggj.apps.googleusercontent.com"; // x.apps.googleusercontent.com
```

Comment:
Something I just realized now is that this information can't be in the git because it'll be different for everyone. That's not related to your changes though because I had it in the git mistakenly too. Maybe we could relocate these lines to the google-client-id file in the private/ directory? Lmk what you think